### DISCUSSION TITLE
Field Order

### DISCUSSION THREAD

**Discussion by davidfilkins** [JoshClose/CsvHelper#2002] [General]
Is it possible to control the order of the columns in created files?

**Comment by JoshClose** [comment]
Yes. Use `Index` in either a mapping or attribute.

**Reply by davidfilkins** [reply]
Thanks for the fast reply! I'll give it a whirl...

**Reply by JoshClose** [reply]
When writing, it uses both `Index` and `Name`. When reading, `Name` will take priority over `Index`.
